ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.Package
+import io.mockk.every
+import io.mockk.mockk
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.Locale
+
+@RunWith(AndroidJUnit4::class)
+class VariableProcessorTest {
+
+    private lateinit var variableDataProvider: VariableDataProvider
+    private lateinit var rcPackage: Package
+    private lateinit var locale: Locale
+
+    @Before
+    fun setUp() {
+        variableDataProvider = mockk()
+        rcPackage = mockk()
+        locale = Locale.ITALY
+    }
+
```

Comment:
You can do it in a follow up, but this could use more tests (like the ones in `VariablesTests`) to check what happens with malformed variables `{{no_spaces}}`, `{{ unknown_variable_names }}`, empty `""` strings, etc.

Reply:
Will add those in a followup PR